### DISCUSSION TITLE
不到最後不鬆懈

### DISCUSSION THREAD

**Discussion by 1104030360** [1101-GS4719-C/Python-Community-GS4719-C#1371] [General]
最近參加了系上的大逃殺，遊戲規則很簡單就是在有限的空間內躲起來不被鬼抓到就贏了，然而如果人被抓到了會變成鬼來抓人，因此最後鬼會越來越多，遊戲難度直線上升，聰明的我在遊戲一開始很快就找到一個堆滿雜物的地方躲起來，在遊戲的過程中有很多鬼經過我旁邊但都沒發現我，我就開始心理膨脹覺得自己在時限內絕對不會被抓到，我甚至直接開啟netflix看起電影，然而就在時間結束前的五分鐘有一個感知能力異常敏銳的鬼開始在我旁邊翻找甚至用手電筒照向我藏的地方，這時的我大氣都不敢喘，深怕被鬼抓到，幸好最後這隻鬼被旁邊跑過去的人吸引並離開我躲的地方，經歷這次的事之後我認知到很多事像是競賽，不到結束都不能鬆懈，因為最後一刻往往是我們最容易被攻破得時候，這也是為什麼常常報紙上會出現大比分逆轉勝的原因，希望各位也能把到手的勝利緊緊握在手中不被偷走！

**Comment by harrycoconut** [comment]
你好👋我是這屆大逃殺的主辦人，首先恭喜你成為我們的獲勝者🥳的確，不到最後一刻真的不能鬆懈，就以我辦大逃殺為例，其實到最後一秒鐘我們還是不斷在找人，甚至延長了三分鐘你們都沒發現，我想這就是急中生智吧😌

**Comment by tower20021202** [comment]
我也有玩那場大逃殺，但我以為只要體能好跑得快就行了，因此我躲避點找得十分隨便，也沒有想好撤退路線，所以在被追殺時就慢慢的被鬼包夾淘汰了😭
